conf/base.config

```diff
@@ -8,16 +8,49 @@ vim: syntax=groovy
 
 
 process {
-    executor = 'local'
-    cpus = { 4 }
-    memory = { 8.GB }
-    time = { 1.h * task.attempt }
-    errorStrategy = { task.exitStatus == 143 ? 'retry' : 'finish' }
-    maxRetries = 0
+    container = params.container
+
+    cpus = { check_max( 16 * task.attempt, 'cpus' ) }
+    memory = { check_max( 256.GB * task.attempt, 'memory' ) }
+    time = { check_max( 120.h * task.attempt, 'time' ) }
+
+    errorStrategy = { task.exitStatus in [1,143,137,104,134,139] ? 'retry' : 'finish' }
+    maxRetries = 3
     maxErrors = '-1'
-    validExitStatus = [0,127]
 
+    withLabel: 'supernova*' {
+        container = params.supernova_container
+    }
```

Comment:
The process `mkoutput` also use the supernova container.

Reply:
Yup - that's why I [renamed it](https://github.com/nf-core/neutronstar/pull/3/files#diff-28b7bea57b04f15beeb4c1de0ee78382R258) 😉

Reply:
Oops, did miss a couple of occurrences of this when renaming though. Will update now.

Reply:
Oh you renamed it. Never mind.